lnrpc/README.md

```diff
@@ -151,17 +151,18 @@ $ export PATH=$PWD/protoc/bin:$PATH
 
 2. Install `golang/protobuf` at commit `ab9f9a6dab164b7d1246e0e688b0ab7b94d8553e`.
 ```bash
-$ git clone https://github.com/golang/protobuf $GOPATH/src/github.com/golang/protobuf
+$ go get -u github.com/golang/protobuf
```

Comment:
This command yields the following error: 
package github.com/golang/protobuf: no Go files in /home/l/gocode/src/github.com/golang/protobuf

Reply:
Yep should be `github.com/golang/protobuf/protoc-gen-go`. Fixed!